crates/core_arch/src/arm_shared/neon/mod.rs

```diff
@@ -2867,11 +3117,8 @@ pub unsafe fn vpmax_f32(a: float32x2_t, b: float32x2_t) -> float32x2_t {
 #[target_feature(enable = "neon")]
 #[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
 #[rustc_legacy_const_generics(1)]
-#[cfg_attr(all(test, target_arch = "arm"), assert_instr("vmov.32", IMM5 = 1))]
-#[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(mov, IMM5 = 1))]
-// Based on the discussion in https://github.com/rust-lang/stdarch/pull/792
-// `mov` seems to be an acceptable intrinsic to compile to
-// #[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(vmov, IMM5 = 1))]
+#[cfg_attr(all(test, target_arch = "arm"), assert_instr("nop", IMM5 = 1))]
+#[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(nop, IMM5 = 1))]
```

Comment:
```suggestion
#[cfg_attr(test, assert_instr(nop, IMM5 = 1))]
```

Reply:
OK, I have made the corresponding changes.
I think I can submit a new PR later to correct all `assert_instr` in arm/aarch64 mod.